meetings/2017-10-25.md

```diff
@@ -0,0 +1,120 @@
+# Node.js Foundation Technical Steering Committee (TSC) Meeting 2017-10-25
+
+## Links
+
+* **Recording**: https://www.youtube.com/watch?v=p1LB6VxKchw
+* **GitHub Issue**: https://github.com/nodejs/TSC/issues/396
+* **Minutes Google Doc**: https://docs.google.com/document/d/1ptEaqEG9sI8EhBFqZv1MQhyuEFfq1F81XyXbMqHyN84/
+
+## Present
+
+* Anna Henningsen @addaleax (TSC)
+* Colin Ihrig @cjihrig (TSC)
+* Jeremiah Senkpiel @Fishrock123 (TSC)
+* James M Snell @jasnell (TSC)
+* Michael Dawson @mhdawson (TSC)
+* Myles Borins @MylesBorins (TSC)
+* Ali Ijaz Sheikh @ofrobots (TSC)
+* Rod Vagg @rvagg (TSC)
+* Trevor Norris @trevnorris (TSC)
+* Rich Trott @Trott (TSC)
+* William Kapke @williamkapke (Node.js Community Board representative)
+* Forrest L Norvell @othiym23 (Moderation team member for moderation update)
+
+## Agenda
+
+## Announcements
+
+* Security release yesterday.  There was an unrelated regression in 8.8.0 release, 8.8.1 release
+   being prepared today. Includes unflagging HTTP2.
+
+### WG update
+
+* Update from the moderation team (Forrest)
+
+* Deck used for presentation https://docs.google.com/presentation/d/1cN712pneaa7L6_wVl9-M2OpK_2Am8EbarYyoJxxyyYA/edit#slide=id.p
+
+* 10 members, so far have had to be intervene in 0 zeros, though there were 2 request
```

Comment:
What is 0 zeros? Also, request -> requests.

Reply:
Fixed, believe if should have been issues.  Added s to request.